web/styles/subscriptions.css

```
@@ -968,6 +968,14 @@ div.settings-radio-input-parent {
     }
 }
 
+.multiuse_radio_section .settings-radio-input-parent {
+    border-bottom: 0;
+
+    input {
```

When compiling frontend, I got a warning on this line:

```
  Module Warning (from ../node_modules/.pnpm/postcss-loader@7.1.0_6puvukfnwbwq425eep7g4z27be/node_modules/postcss-loader/dist/cjs.js):
  Warning

  (974:5) postcss-nesting: Nested selectors must start with a symbol and "input" begins with a letter.

```

Adding a current selector before input fixed the warning for me:
```suggestion
    & input {
```